requirements/edx/base.txt

```diff
@@ -97,7 +97,7 @@ edx-celeryutils==0.5.1    # via -r requirements/edx/base.in, super-csv
 edx-completion==3.2.4     # via -r requirements/edx/base.in
 edx-django-release-util==0.4.4  # via -r requirements/edx/base.in
 edx-django-sites-extensions==2.5.1  # via -r requirements/edx/base.in
-edx-django-utils==3.6.0   # via -r requirements/edx/base.in, django-config-models, edx-drf-extensions, edx-enterprise, edx-rest-api-client, edx-when
+-e git+https://github.com/edx/edx-django-utils.git@msingh/add/fix#egg=edx-django-utils   # via -r requirements/edx/base.in, django-config-models, edx-drf-extensions, edx-enterprise, edx-rest-api-client, edx-when
```

Comment:
@ reviewer I will remove this link once both this PR and edx-django-utils PR is ready to be merged. I'm leaving it so that any change made based on reviewer comments can be tested easily.